src/libraries/System.Linq.Expressions/tests/New/NewWithByRefParameterTests.cs

```diff
@@ -70,7 +70,8 @@ public void CreateByRefAliasingInterpreted()
             CreateByRefAliasing(useInterpreter: true);
         }
 
-        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsNotLinqExpressionsBuiltWithIsInterpretingOnly))]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/19286", typeof(PlatformDetection), nameof(PlatformDetection.IsLinqExpressionsBuiltWithIsInterpretingOnly))]
```

Comment:
Switched this one to ActiveIssue - the preceding test has the motivation for this.